frontend/src/components/VImageCell/VImageCell.vue

```diff
@@ -172,6 +172,8 @@ export default defineComponent({
         provider: props.image.provider,
         query: props.searchTerm || "",
         relatedTo: props.relatedTo,
+        sensitivities: props.image.sensitivity?.join(",") ?? "",
```

Comment:
Would it be possible to send an array, not a string? There are many combinations of sensitivity values possible, so it would be difficult to compare them.

We could also send a single `isSensitive` value? It would be less specific, but it would be easier to compare.
Using something like a binary flag for sensitivities would make it easier to compare (e.g. USER_REPORTED, PROVIDER_SUPPLIED, TEXT_FILTERED is 111,   PROVIDER_SUPPLIED, TEXT_FILTERED - 011) but I guess it's making this unnecessarily complex 😆

Reply:
A binary would be functionally identical to a comma-separated concatenated string, provided we sort sensitivities into a fixed order. That can be handled in the media decode step: a523696.

<img width="569" alt="Screenshot 2023-09-07 at 10 40 52 AM" src="https://github.com/WordPress/openverse/assets/16580576/7851ba63-04dd-48d0-8572-8b6dfab29b17">